curp/tests/common/curp_group.rs

```diff
@@ -164,19 +163,92 @@ impl CurpGroup {
         debug!("successfully start group");
         Self {
             nodes,
-            all,
             storage_path,
         }
     }
 
+    pub async fn run_node(
```

Comment:
What's the main difference between `run_node` and `new` in CurpGroup?

Reply:
`new` will create an initialized cluster, `run_node` will run a new node and insert to current cluster